### PR TITLE
CHE-7577 CHE6 UI refinements

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
### What does this PR do?
Improves UI widgets for dark theme.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7577

preferences

revert commit

search

### CODE REVIEW

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/factory/json/ImportFromConfigViewImpl.java [hunk 1]
```diff
@@ -50,24 +50,29 @@ public ImportFromConfigViewImpl(
...
-            "import-from-config-btn-cancel",
-            event -> delegate.onCancelClicked());
-    addButtonToFooter(btnCancel);
-
     buttonImport =
         createButton(
             locale.importButton(),
             "import-from-config-btn-import",
             event -> delegate.onImportClicked());
+    buttonImport.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/factory/json/ImportFromConfigViewImpl.java]: Take a look on other three methods `createButton` in Window class, there is another method which provide _Primary_ style and custom alignment.

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/resources/selector/SelectPathViewImpl.java [hunk 2]
```diff
@@ -153,6 +153,8 @@ public void onClick(ClickEvent event) {
             hide();
           }
         });
+    submitBtn.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/resources/selector/SelectPathViewImpl.java]: Take a look on above comment

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/upload/file/UploadFileViewImpl.java [hunk 3]
```diff
@@ -56,36 +56,40 @@ public UploadFileViewImpl(
...
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onUploadClicked();
               }
             });
-    addButtonToFooter(btnCancel);
 
-    btnUpload =
+    btnUpload.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/upload/file/UploadFileViewImpl.java]: Take a look on above comment

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/upload/folder/UploadFolderFromZipViewImpl.java [hunk 4]
```diff
@@ -59,37 +59,41 @@ public UploadFolderFromZipViewImpl(
...
             new ClickHandler() {
 
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onUploadClicked();
               }
             });
-    addButtonToFooter(btnCancel);
+    btnUpload.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/upload/folder/UploadFolderFromZipViewImpl.java]: Take a look on above comment

File: ide/che-core-ide-ui/src/main/resources/org/eclipse/che/ide/ui/cellview/cellTable.css [hunk 5]
```diff
@@ -100,15 +114,15 @@
 }
 
 .cellTableEvenRowCell {
-    border: 1px solid cellEvenRow;
+    /*border: 1px solid cellEvenRow;*/
```
Comment (vzhukovs) [inline ide/che-core-ide-ui/src/main/resources/org/eclipse/che/ide/ui/cellview/cellTable.css]: Remove commented code

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/checkout/CheckoutReferenceViewImpl.java [hunk 6]
```diff
@@ -48,31 +48,32 @@ public CheckoutReferenceViewImpl(GitLocalizationConstant locale) {
...
             new ClickHandler() {
 
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onCheckoutClicked(reference.getValue());
               }
             });
-    addButtonToFooter(btnCancel);
+    btnCheckout.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/checkout/CheckoutReferenceViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/CompareViewImpl.java [hunk 7]
```diff
@@ -92,22 +92,25 @@ public CompareViewImpl(
             locale.buttonSaveChanges(),
             "git-compare-save-changes-btn",
             event -> delegate.onSaveChangesClicked());
+    btnSaveChanges.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/CompareViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/fetch/FetchViewImpl.java [hunk 8]
```diff
@@ -76,31 +76,32 @@ protected FetchViewImpl(GitResources resources, GitLocalizationConstant locale)
...
             new ClickHandler() {
 
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onFetchClicked();
               }
             });
-    addButtonToFooter(btnCancel);
+    btnFetch.addStyleName(super.resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/fetch/FetchViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/history/HistoryViewImpl.java [hunk 9]
```diff
@@ -195,29 +195,30 @@ public void onDoubleClick(DoubleClickEvent event) {
...
+            "git-history-compare",
             new ClickHandler() {
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCloseClicked();
+                delegate.onCompareClicked();
               }
             });
-    addButtonToFooter(btnClose);
+    btnCompare.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/history/HistoryViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/merge/MergeViewImpl.java [hunk 10]
```diff
@@ -156,31 +156,32 @@ public void onKeyboard(KeyboardEvent event) {
...
             new ClickHandler() {
 
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onMergeClicked();
               }
             });
-    addButtonToFooter(btnCancel);
+    btnMerge.addStyleName(super.resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/merge/MergeViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/pull/PullViewImpl.java [hunk 11]
```diff
@@ -74,31 +74,32 @@ protected PullViewImpl(GitResources resources, GitLocalizationConstant locale) {
...
             new ClickHandler() {
 
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onPullClicked();
               }
             });
-    addButtonToFooter(btnCancel);
+    btnPull.addStyleName(super.resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/pull/PullViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/push/PushToRemoteViewImpl.java [hunk 12]
```diff
@@ -70,31 +70,32 @@ protected PushToRemoteViewImpl(GitResources resources, GitLocalizationConstant l
...
             new ClickHandler() {
 
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onPushClicked();
               }
             });
-    addButtonToFooter(btnCancel);
+    btnPush.addStyleName(super.resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/push/PushToRemoteViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/remote/RemoteViewImpl.java [hunk 13]
```diff
@@ -110,6 +97,7 @@ public void onClick(ClickEvent event) {
                 delegate.onAddClicked();
               }
             });
+    btnAdd.addStyleName(super.resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/remote/RemoteViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/reset/commit/ResetToCommitViewImpl.java [hunk 14]
```diff
@@ -91,31 +91,32 @@ protected ResetToCommitViewImpl(GitResources resources, GitLocalizationConstant
...
             new ClickHandler() {
 
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onResetClicked();
               }
             });
-    addButtonToFooter(btnCancel);
+    btnReset.addStyleName(super.resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/reset/commit/ResetToCommitViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/reset/files/ResetFilesViewImpl.java [hunk 15]
```diff
@@ -79,31 +79,32 @@ protected ResetFilesViewImpl(GitLocalizationConstant locale, GitResources resour
...
             new ClickHandler() {
 
               @Override
               public void onClick(ClickEvent event) {
-                delegate.onCancelClicked();
+                delegate.onResetClicked();
               }
             });
-    addButtonToFooter(btnCancel);
+    btnReset.addStyleName(super.resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/reset/files/ResetFilesViewImpl.java]: Take a look on above comments

File: plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/revert/RevertCommitViewImpl.java [hunk 16]
```diff
@@ -162,23 +162,24 @@ public String getValue(Revision revision) {
...
-    addButtonToFooter(btnCancel);
-
     btnRevert =
         createButton(
             locale.buttonRevert(),
             "git-revert",
             event -> {
               delegate.onRevertClicked();
             });
+    btnRevert.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/revert/RevertCommitViewImpl.java]: Take a look on above comments

File: plugins/plugin-help/che-plugin-help-ext-client/src/main/java/org/eclipse/che/ide/ext/help/client/about/AboutViewImpl.java [hunk 17]
```diff
@@ -76,6 +76,7 @@ public void onClick(ClickEvent event) {
                 delegate.onOkClicked();
               }
             });
+    btnOk.addStyleName(resources.windowCss().primaryButton());
```
Comment (vzhukovskii) [inline plugins/plugin-help/che-plugin-help-ext-client/src/main/java/org/eclipse/che/ide/ext/help/client/about/AboutViewImpl.java]: Take a look on above comments

**Review by vzhukovs — CHANGES_REQUESTED** [review]

File: ide/che-core-ide-ui/src/main/resources/org/eclipse/che/ide/ui/cellview/cellTable.css (hunk 5 above)
Comment (vitaliy-guliy) [reply ide/che-core-ide-ui/src/main/resources/org/eclipse/che/ide/ui/cellview/cellTable.css]: Done

**Comment by vitaliy-guliy** [comment]
ci-build

**Comment by vitaliy-guliy** [comment]
ci-test

**Review by vzhukovs — APPROVED** [review]

**Comment by vitaliy-guliy** [comment]
ci-build

**Comment by vitaliy-guliy** [comment]
ci-build

**Comment by vitaliy-guliy** [comment]
ci-test

**Review by ashumilova — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Review by SkorikSergey — APPROVED** [review]

**Pull request merged**